### DISCUSSION TITLE
1.5.0rc4

### DISCUSSION THREAD

**Discussion by villebro** [apache/superset#19833] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.5.0 Here’s the list of
new cherries since the previous release candidate:

- #19776: feat: add renameOperator (@zhaoyongjie)
- #19448 fix(sql lab): when editing a saved query, the status is lost when switching tabs (@diegomedina248)
- #19806 fix(plugin-chart-table): Fix display of column config in table chart (@kgabryje)
- #19802 fix: lost renameOperator in mixed timeseries chart (@zhaoyongjie)
- #19765 fix(chart & explore): Show labels for `SliderControl` (@prosdev0107)
- #16619 fix: dashboard standalone class not added when parameter set (@trepmag)
- #19783 fix: SQL Lab UI Error: Objects are not valid as a React child (@diegomedina248)
- #19370 fix(sql lab): replace the output column in the query history table (@diegomedina248)
- #19772 fix(dashboard): copy permalink to dashboard chart (@villebro)
- #19603 fix(explore): make to show the null value as N/A in view result (@prosdev0107)
- #19749 fix(permalink): remove memoize on get salt func (@villebro)
- #19698 fix: Filter dependencies are not being applied in default values (@michael-s-molina)
- #19732 chore: Clean redundant dependency from useMemo dep array (@kgabryje)

The release notes (unchanged since rc3):
https://github.com/apache/superset/tree/master/RELEASING/release-notes-1-5

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.0rc4/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.0rc4

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.0rc4/CHANGELOG.md

The Updating instructions for the release (unchanged since rc3):
https://github.com/apache/superset/blob/1.5.0rc4/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.5.0rc4'>1.5.0rc4</a>.</em>

**Comment by zhaoyongjie** [comment]
+1 (binding)

On Mon, Apr 25, 2022 at 4:56 PM Ville Brofeldt ***@***.***>
wrote:

> Hello Superset Community,
>
> This is a call for the vote to release Apache Superset version 1.5.0
> Here’s the list of
> new cherries since the previous release candidate:
>
>    - #19776 <https://github.com/apache/superset/pull/19776>: feat: add
>    renameOperator ***@***.*** <https://github.com/zhaoyongjie>)
>    - #19448 <https://github.com/apache/superset/pull/19448> fix(sql lab):
>    when editing a saved query, the status is lost when switching tabs (
>    @diegomedina248 <https://github.com/diegomedina248>)
>    - #19806 <https://github.com/apache/superset/pull/19806>
>    fix(plugin-chart-table): Fix display of column config in table chart (
>    @kgabryje <https://github.com/kgabryje>)
>    - #19802 <https://github.com/apache/superset/pull/19802> fix: lost
>    renameOperator in mixed timeseries chart ***@***.***
>    <https://github.com/zhaoyongjie>)
>    - #19765 <https://github.com/apache/superset/pull/19765> fix(chart &
>    explore): Show labels for SliderControl ***@***.***
>    <https://github.com/prosdev0107>)
>    - #16619 <https://github.com/apache/superset/pull/16619> fix:
>    dashboard standalone class not added when parameter set ***@***.***
>    <https://github.com/trepmag>)
>    - #19783 <https://github.com/apache/superset/pull/19783> fix: SQL Lab
>    UI Error: Objects are not valid as a React child ***@***.***
>    <https://github.com/diegomedina248>)
>    - #19370 <https://github.com/apache/superset/pull/19370> fix(sql lab):
>    replace the output column in the query history table ***@***.***
>    <https://github.com/diegomedina248>)
>    - #19772 <https://github.com/apache/superset/pull/19772>
>    fix(dashboard): copy permalink to dashboard chart ***@***.***
>    <https://github.com/villebro>)
>    - #19603 <https://github.com/apache/superset/pull/19603> fix(explore):
>    make to show the null value as N/A in view result ***@***.***
>    <https://github.com/prosdev0107>)
>    - #19749 <https://github.com/apache/superset/pull/19749>
>    fix(permalink): remove memoize on get salt func ***@***.***
>    <https://github.com/villebro>)
>    - #19698 <https://github.com/apache/superset/pull/19698> fix: Filter
>    dependencies are not being applied in default values ***@***.***
>    <https://github.com/michael-s-molina>)
>    - #19732 <https://github.com/apache/superset/pull/19732> chore: Clean
>    redundant dependency from useMemo dep array ***@***.***
>    <https://github.com/kgabryje>)
>
> The release notes (unchanged since rc3):
> https://github.com/apache/superset/tree/master/RELEASING/release-notes-1-5
>
> The release candidate:
> https://dist.apache.org/repos/dist/dev/superset/1.5.0rc4/
>
> Git tag for the release:
> https://github.com/apache/superset/tree/1.5.0rc4
>
> The Change Log for the release:
> https://github.com/apache/superset/blob/1.5.0rc4/CHANGELOG.md
>
> The Updating instructions for the release (unchanged since rc3):
> https://github.com/apache/superset/blob/1.5.0rc4/UPDATING.md
>
> public keys are available at:
>
> https://www.apache.org/dist/superset/KEYS
>
> The vote will be open for at least 72 hours or until the necessary number
> of votes are reached.
>
> Please vote accordingly:
>
> [ ] +1 approve
> [ ] +0 no opinion
> [ ] -1 disapprove with the reason
>
> Thanks,
> The Apache Superset Team
> ------------------------------
> *This discussion was created from the release 1.5.0rc4
> <https://github.com/apache/superset/releases/tag/1.5.0rc4>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/superset/discussions/19833>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAPMKUUXZ5TAB5ZIONXHIXTVGZM3BANCNFSM5UH3QE5Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Best regards,

Yongjie
